modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -86,15 +86,15 @@ namespace
                     return gim.metadata(nh).get<NodeKind>().k == NodeKind::ISLAND;
                 });
 
-            const auto out_rois = cv::gimpl::getCompileArg<cv::GFluidOutputRois>(args);
```

Comment:
So the old name is not available anymore?

I'd keep the old function `cv::gimpl`, it could be implemented via the new `cv::gapi` one.

If the `cv::gimpl` has gone, we have all our private backends broken now.

Reply:
I see this issue fixed, thanks